src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        final Colors randomColor = ColorSupplier.getRandomColor();
+        final Random randomNumber = new Random();
+        int randomBall = randomNumber.nextInt(101);
```

Comment:
this index should be a constant as well (for example `private static final int MAX_NUMBER`)